### PR TITLE
website: content updates for developer

### PR DESCRIPTION
**Pull request by BRKalow**
- Fix links pointing at redirects
- Remove textual references to HashiCorp Learn

### CODE REVIEW

File: website/Makefile [hunk 1]
```diff
@@ -16,7 +16,9 @@ DOCKER_RUN_FLAGS=-it \
 		--volume "$(PWD)/redirects.js:/app/redirects.js" \
 		--volume "next-dir:/app/website-preview/.next" \
 		--volume "$(PWD)/.env:/app/.env" \
-		-e "REPO=nomad"
+		-e "REPO=nomad" \
+		-e "PREVIEW_MODE=developer"
```
Comment (BRKalow) [inline website/Makefile:20]:
```suggestion
		-e "PREVIEW_MODE=io"
```

File: website/data/docs-nav-data.json [hunk 2]
```diff
@@ -23,11 +23,11 @@
           },
           {
             "title": "Reference Architecture",
-            "href": "https://learn.hashicorp.com/tutorials/nomad/production-reference-architecture-vm-with-consul"
+            "href": "/nomad/tutorials/enterprise/production-reference-architecture-vm-with-consul"
```
Comment (BRKalow) [inline website/data/docs-nav-data.json:26]:
```suggestion
            "href": "https://learn.hashicorp.com/tutorials/nomad/production-reference-architecture-vm-with-consul"
```

File: website/data/docs-nav-data.json [hunk 3]
```diff
@@ -23,11 +23,11 @@
...
           },
           {
             "title": "Deployment Guide",
-            "href": "https://learn.hashicorp.com/tutorials/nomad/production-deployment-guide-vm-with-consul"
+            "href": "/nomad/tutorials/enterprise/production-deployment-guide-vm-with-consul"
```
Comment (BRKalow) [inline website/data/docs-nav-data.json:30]:
```suggestion
            "href": "https://learn.hashicorp.com/tutorials/nomad/production-deployment-guide-vm-with-consul"
```

File: website/data/docs-nav-data.json [hunk 4]
```diff
@@ -1993,15 +2003,15 @@
       },
       {
         "title": "Cluster Management",
-        "href": "https://learn.hashicorp.com/collections/nomad/manage-clusters"
+        "href": "/nomad/tutorials/manage-clusters"
```
Comment (BRKalow) [inline website/data/docs-nav-data.json:2006]:
```suggestion
        "href": "https://learn.hashicorp.com/collections/nomad/manage-clusters"
```

File: website/data/docs-nav-data.json [hunk 5]
```diff
@@ -1993,15 +2003,15 @@
...
       },
       {
         "title": "Transport Security",
-        "href": "https://learn.hashicorp.com/collections/nomad/transport-security"
+        "href": "/nomad/tutorials/transport-security"
```
Comment (BRKalow) [inline website/data/docs-nav-data.json:2010]:
```suggestion
        "href": "https://learn.hashicorp.com/collections/nomad/transport-security"
```

File: website/data/docs-nav-data.json [hunk 6]
```diff
@@ -1993,15 +2003,15 @@
...
       },
       {
         "title": "Access Control",
-        "href": "https://learn.hashicorp.com/collections/nomad/access-control"
+        "href": "/nomad/tutorials/access-control"
```
Comment (BRKalow) [inline website/data/docs-nav-data.json:2014]:
```suggestion
        "href": "https://learn.hashicorp.com/collections/nomad/access-control"
```

File: website/data/intro-nav-data.json [hunk 7]
```diff
@@ -29,27 +29,23 @@
     "routes": [
       {
         "title": "Overview",
-        "href": "https://learn.hashicorp.com/collections/nomad/get-started"
+        "href": "/nomad/tutorials/get-started"
```
Comment (BRKalow) [inline website/data/intro-nav-data.json:32]:
```suggestion
        "href": "https://learn.hashicorp.com/collections/nomad/get-started"
```

File: website/data/intro-nav-data.json [hunk 8]
```diff
@@ -29,27 +29,23 @@
...
       },
       {
         "title": "Running Nomad",
-        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-run"
+        "href": "/nomad/tutorials/get-started/get-started-run"
```
Comment (BRKalow) [inline website/data/intro-nav-data.json:36]:
```suggestion
        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-run"
```

File: website/data/intro-nav-data.json [hunk 9]
```diff
@@ -29,27 +29,23 @@
...
       },
       {
         "title": "Jobs",
-        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-jobs"
-      },
-      {
-        "title": "Clustering",
-        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-cluster"
+        "href": "/nomad/tutorials/get-started/get-started-jobs"
```
Comment (BRKalow) [inline website/data/intro-nav-data.json:40]:
```suggestion
        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-jobs"
```

File: website/data/intro-nav-data.json [hunk 10]
```diff
@@ -29,27 +29,23 @@
...
       },
       {
         "title": "Web UI",
-        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-ui"
+        "href": "/nomad/tutorials/get-started/get-started-ui"
```
Comment (BRKalow) [inline website/data/intro-nav-data.json:44]:
```suggestion
        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-ui"
```

File: website/data/intro-nav-data.json [hunk 11]
```diff
@@ -29,27 +29,23 @@
...
       },
       {
         "title": "Next Steps",
-        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-learn-more"
+        "href": "/nomad/tutorials/get-started/get-started-learn-more"
```
Comment (BRKalow) [inline website/data/intro-nav-data.json:48]:
```suggestion
        "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-learn-more"
```

File: website/scripts/website-build.sh [hunk 12]
```diff
@@ -6,6 +6,8 @@ PREVIEW_DIR=website-preview
 CLONE_DIR=website-preview
 # The product for which we are building the deploy preview
 PRODUCT=nomad
+# Preview mode, controls the UI rendered (either the product site or developer). Can be `io` or `developer`
+PREVIEW_MODE=developer
```
Comment (BRKalow) [inline website/scripts/website-build.sh:10]:
```suggestion
PREVIEW_MODE=io
```

File: website/content/docs/upgrade/upgrade-specific.mdx [hunk 13]
```diff
@@ -1485,7 +1485,7 @@ deleted and then Nomad 0.3.0 can be launched.
 [cap_drop_exec]: /docs/drivers/exec#cap_drop
 [`log_file`]: /docs/configuration#log_file
 [Upgrading to Raft Protocol 3]: /docs/upgrade#upgrading-to-raft-protocol-3
-[`Local`]: https://www.consul.io/docs/security/acl/acl-tokens#token-attributes
+[`Local`]: https://www.consul.io/docs/security/acl/acl-tokens#token-attributes#token-attributes
```
Comment (BRKalow) [inline website/content/docs/upgrade/upgrade-specific.mdx:1488]:
```suggestion
[`Local`]: https://www.consul.io/docs/security/acl/acl-tokens#token-attributes
```

File: website/content/docs/install/index.mdx [hunk 14]
```diff
@@ -9,7 +9,7 @@ description: Learn how to install Nomad.
 Nomad is available as a pre-compiled binary or as a package for several
 operating systems. You can also [build Nomad from source](#from-source).
 
--> If you are interested in trialing Nomad without installing it locally, see the
+\-> If you are interested in trialing Nomad without installing it locally, see the
```
Comment (BRKalow) [inline website/content/docs/install/index.mdx:12]:
```suggestion
-> If you are interested in trialing Nomad without installing it locally, see the
```

File: website/content/docs/install/index.mdx [hunk 15]
```diff
@@ -33,7 +33,7 @@ The output is a list of locations separated by colons. You can make Nomad
 available by moving the binary to one of the listed locations, or by adding
 Nomad's location to your `PATH`.
 
--> **Tip (Linux-based or Mac):** Permanently add a new location to your path by
+\-> **Tip (Linux-based or Mac):** Permanently add a new location to your path by
```
Comment (BRKalow) [inline website/content/docs/install/index.mdx:36]:
```suggestion
-> **Tip (Linux-based or Mac):** Permanently add a new location to your path by
```

File: website/content/docs/install/index.mdx [hunk 16]
```diff
@@ -42,7 +42,7 @@ binary to that list and save the file. Then reload your shell's configuration
 with the command `source ~/.bashrc`, replacing `bash` with the name of your
 shell.
 
--> **Tip (Windows):** Add a location to your path via the GUI by navigating to
+\-> **Tip (Windows):** Add a location to your path via the GUI by navigating to
```
Comment (BRKalow) [inline website/content/docs/install/index.mdx:45]:
```suggestion
-> **Tip (Windows):** Add a location to your path via the GUI by navigating to
```

File: website/content/docs/install/index.mdx [hunk 17]
```diff
@@ -140,7 +140,7 @@ $ sudo yum -y install nomad
 </Tab>
 </Tabs>
 
--> **TIP:** Now that you have added the HashiCorp repository, you can install
+\-> **TIP:** Now that you have added the HashiCorp repository, you can install
```
Comment (BRKalow) [inline website/content/docs/install/index.mdx:143]:
```suggestion
-> **TIP:** Now that you have added the HashiCorp repository, you can install
```

File: website/content/docs/install/index.mdx [hunk 18]
```diff
@@ -191,7 +191,7 @@ installation.
 </Tab>
 </Tabs>
 
----
+- - -
```
Comment (BRKalow) [inline website/content/docs/install/index.mdx:194]:
```suggestion
---
```

File: website/content/docs/install/index.mdx [hunk 19]
```diff
@@ -232,7 +232,7 @@ Other commands:
     version         Prints the Nomad version
 ```
 
----
+- - -
```
Comment (BRKalow) [inline website/content/docs/install/index.mdx:235]:
```suggestion
---
```

File: website/content/intro/index.mdx [hunk 20]
```diff
@@ -53,4 +53,4 @@ comparisons.
 
 ## Next Steps
 
-See the [Use Cases](/intro/use-cases) and [Who Uses Nomad](/intro/who-uses-nomad) page to understand the many ways Nomad is used in production today across many industries to solve critical, real-world business objectives.
+See the [Use Cases](/intro/use-cases) and [Who Uses Nomad](/docs/who-uses/noamd) page to understand the many ways Nomad is used in production today across many industries to solve critical, real-world business objectives.
```
Comment (tunzor) [inline website/content/intro/index.mdx:56]: The `Who Uses Nomad` page no longer exists in the newer versions.
```suggestion
See the [Use Cases](/intro/use-cases) page to understand the many ways Nomad is used in production today across many industries to solve critical, real-world business objectives.
```

File: website/data/intro-nav-data.json [hunk 21]
```diff
@@ -39,10 +39,6 @@
         "title": "Jobs",
         "href": "https://learn.hashicorp.com/tutorials/nomad/get-started-jobs"
       },
-      {
```
Comment (tunzor) [inline website/data/intro-nav-data.json:42]: Think this should be:
```
{
    "title": "Clustering",
    "href": "https://learn.hashicorp.com/tutorials/nomad/clustering"
},
```

**Review by tunzor — CHANGES_REQUESTED** [review]
Couple snags from the script to fix (adding extra hash links and trying to escape characters) and a few link fixes.

File: website/content/docs/configuration/vault.mdx [hunk 22]
```diff
@@ -60,13 +60,13 @@ vault {
 
 - `cert_file` `(string: "")` - Specifies the path to the certificate used for
   Vault communication. This must be set if
-  [tls_require_and_verify_client_cert](https://www.vaultproject.io/docs/configuration/listener/tcp#tls_require_and_verify_client_cert)
+  [tls_require_and_verify_client_cert](https://www.vaultproject.io/docs/configuration/listener/tcp#tls_require_and_verify_client_cert#tls_require_and_verify_client_cert)
```
[the remark this answers is not in the document]
Comment (BRKalow) [reply website/content/docs/configuration/vault.mdx:63]: Oops, good catch!

**Review by tunzor — APPROVED** [review]

**Pull request merged**